request/builder.js

```diff
@@ -371,6 +371,7 @@ class ContractExecutionRequestBuilder {
   constructor(request, signer) {
     this.request = request;
     this.signer = signer;
+    this.functionArgument = '';
```

Comment:
Should it be `null` in the above case ?

Reply:
Sorry I made a mistake here.
I was thinking to use `'{}'` in case that the server side doesn't handle nullable value.
I will remove this line, then the value of function_argument in gRPC message will be null if there's no value for it.